src/pages/community/members.js

```diff
@@ -206,7 +215,7 @@ const MembersPage = () => {
         <Dropdown options={options} handleChange={handleChange} />
 
         {!members ? (
-          <MultipleMembers members={[options[10]]} />
+          <MultipleMembers members={[options[11]]} />
```

Comment:
We need to get away with this fixed number-based approach and better use the array length or some other way such that every time we don't need to update the size.
@chinu-anand do you mind fixing it along in this PR?